java/ql/src/semmle/code/java/frameworks/spring/SpringErrors.qll

```diff
@@ -0,0 +1,22 @@
+/** Definitions of flow steps through utility methods of `org.springframework.validation.Errors`. */
+
+import java
+private import semmle.code.java.dataflow.ExternalFlow
```

Comment:
This needs bidirectional import. E.g. by importing `Spring.qll` in `ExternalFlow.qll`.